validphys2/src/validphys/scripts/vp_get.py

```diff
@@ -51,12 +49,11 @@ def main():
         sys.exit(1)
     p.add_argument('resource_type', help="Type of the resource to be obtained. "
                    "See --list for a list of resource types.")
-    p.add_argument('resource_name', help="Identifier of the resource.")
+    p.add_argument('resource_name', help="Identifier of the resource.", nargs='+')
```

Comment:
Please, add a line below `parse_args` so that it fails if `len(resource_name) > 1` and `resource_type != photon`